helix-core/src/test/java/org/apache/helix/integration/TestDisableCustomCodeRunner.java

```diff
@@ -209,9 +214,7 @@ public void test() throws Exception {
 
     // Re-enable custom-code runner
     admin.enableResource(clusterName, customCodeRunnerResource, true);
-    result = ClusterStateVerifier.verifyByZkCallback(
-        new ClusterStateVerifier.BestPossAndExtViewZkVerifier(ZK_ADDR, clusterName));
-    Assert.assertTrue(result);
+    Assert.assertTrue(verifier.verifyByPolling());
```

Comment:
Just want to make sure, besides waiting for some time before verifying, the verify by polling function is same as what previous is done?

Reply:
The new one BestPossibleExternalViewVerifier replaced the deprecated one ClusterStateVerifier and BestPossAndExtViewZkVerifier